### PR TITLE
docs(adapters): update readme version badges -> canary

### PR DESCRIPTION
**Pull request by ndom91**
## Reasoning 💡

Update README badges to point to latest canary version until #68 is complete.

## Checklist 🧢

- [x] Documentation
- ~[ ] Tests~
- [x] Ready to be merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,14 +19,14 @@ This repository is a lerna mono-repo which is home to multiple next-auth adapter
 
 Currently, there are the following adapters ready for use
```
Comment (ndom91) [inline README.md:20]: Maybe also change the text here until #68 is complete to explicitly recommend / warn to use the canary versions? @balazsorban44

Comment (balazsorban44) [reply README.md:20]: Yes, I think we should say something like these are experimental adapters, and we would like feedback on them!

**Review by balazsorban44 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ndom91) [reply README.md:20]: Alright, updated! What do you think? 

I tried to make it clear what the plan is regarding the legacy  adapters as well.

**Review by balazsorban44 — APPROVED** [review]
Much better!

**Pull request merged**